suchak1/intravideo_search
# Finished out the tests for constructor and render for GUI

***For All Submissions:***


*Proposed Changes:*

Finished writing all the tests for the GUI constructor as well as render().


*Testing Instructions:*

* [ ] Does simply running `pytest` suffice?
I could not get pytest to run on my personal machine, but I think that has something to do with my machine. It should run with pytest on a different machine.

**Finally**,
* [X] Have you updated the version number, by running `git tag v0.x`?

------

**For new features:**
* [ ] If you have used new packages, have you remade `requirements.txt` (explained in README)?

**For new tests:**
* [X] Have you made sure your tests are in the `tests/` dir?
* [X] Have you made sure your tests are in a file that starts with '`test_`'?
* [X] Have you made sure your test functions starts with '`test_`'?
